django_ipam/base/models.py

```diff
@@ -34,14 +34,27 @@ def clean(self):
             if self.id != subnet["id"] and ip_network(self.subnet).overlaps(subnet["subnet"]):
                 raise ValidationError({'subnet': _('Subnet overlaps with %s') % (subnet["subnet"])})
 
-    def request_ip(self):
+    def get_first_available_ip(self):
         subnet = get_object_or_404(swapper.load_model("django_ipam", "Subnet"), pk=self.id)
         ipaddress_set = [ipaddr.ip_address for ipaddr in subnet.ipaddress_set.all()]
```

Comment:
same here, simply use `self.ipaddress_set.all()` and get rid of theprevious line, I don't know why I didn't notice this before